.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --workspace --all-targets --features opentelemetry_0_15
+          args: --workspace --all-targets --features opentelemetry_0_16
```

Comment:
It'd be ideal to run tests for each version of OTEL - can you amend?

Reply:
I added a matrix to the jobs that used `--features`. Not sure it's really needed on the other jobs besides test, but they have it now.

Test run here: https://github.com/wolverian/reqwest-middleware/actions/runs/1181732588

Reply:
Better safe than sorry, thanks!